Library/Homebrew/utils/pypi.rb

```diff
@@ -58,17 +50,38 @@ def get_pypi_info(package, version)
   end
 
   # Return true if resources were checked (even if no change).
-  def update_python_resources!(formula, version = nil, print_only: false, silent: false,
-                               ignore_non_pypi_packages: false)
-
-    if !print_only && AUTOMATIC_RESOURCE_UPDATE_BLOCKLIST.include?(formula.full_name)
-      odie "The resources for \"#{formula.name}\" need special attention. Please update them manually."
-      return
+  def update_python_resources!(formula, version: nil, package_name: nil, extra_packages: nil, exclude_packages: nil,
+                               print_only: false, silent: false, ignore_non_pypi_packages: false)
+
+    auto_update_list = formula.tap.audit_exceptions[:automatic_resource_update_list]
+    if package_name.blank? && extra_packages.blank? && !print_only &&
+       auto_update_list.present? && auto_update_list.key?(formula.full_name)
+
+      list_entry = auto_update_list[formula.full_name]
+      case list_entry
+      when false
+        odie "The resources for \"#{formula.name}\" need special attention. Please update them manually."
+      when String
+        package_name = list_entry
+      when Hash
+        package_name = list_entry["package_name"]
+        extra_packages = list_entry["extra_packages"]
+        exclude_packages = list_entry["exclude_packages"]
+      end
     end
 
-    pypi_name = url_to_pypi_package_name formula.stable.url
+    package_name ||= url_to_pypi_package_name formula.stable.url
+    version ||= formula.version
+    extra_packages ||= []
+    exclude_packages ||= []
+
+    # opoo "package_name: #{package_name}"
+    # opoo "version: #{version}"
+    # opoo "extra_packages: #{extra_packages}"
+    # opoo "exclude_packages: #{exclude_packages}"
+    # odie ""
```

Comment:
```suggestion
```

Reply:
Ugh... I even wrote myself a note to remove these 🤦‍♂️